net/libndp/Makefile

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libndp
+PKG_VERSION:=1.6
+PKG_RELEASE:=1
+PKG_LICENSE:=LGPL-2.1
+PKG_MAINTAINER:=Thomas Guyot-Sionnest <tguyot@gmail.com>
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://github.com/jpirko/libndp/archive/
+PKG_HASH:=565d6c4167f83ec697c762ea002f23e8f0b00828d0749b1ce928f068543e5aad
+
+
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+PKG_FIXUP:=autoreconf
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/libndp
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=NDP Library/Tool
+  URL:=https://github.com/jpirko/libndp/libndp-1.6/
+endef
+
+define Package/libndp/description
+    Library for Neighbor Discovery Protocol
+endef
+
+define Build/Configure
+	(cd $(PKG_BUILD_DIR); ./autogen.sh );
+	$(call Build/Configure/Default)
+endef
+
+define Package/libndp/install
+	$(INSTALL_DIR) $(1)/usr/lib/
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libndp.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/ndptool $(1)/usr/bin/
```

Comment:
No, this needs to be defined as a separate package, you may very well use the same Makefile however.

Reply:
https://github.com/openwrt/packages/blob/master/sound/lame/Makefile
If you need a template

Reply:
Ok. Can't fix that right now, will update the PR in a few days and test. Thanks for reviewing quickly!